.agents/rules/skills.md

```
@@ -0,0 +1,63 @@
+# Skill Rules
+
+Follow these rules when creating or modifying skills.
+
+## Specification
+
+Conform to the [Agent Skills Specification](https://agentskills.io/specification.md).
+
+### Directory Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)